lvis/results.py

```diff
@@ -7,14 +7,28 @@
 
 
 class LVISResults(LVIS):
-    def __init__(self, lvis_gt, results, max_dets=300, precompute_boundary=False, dilation_ratio=0.02):
+    def __init__(
+        self,
+        lvis_gt,
+        results,
+        max_dets_per_class=-1,
```

Comment:
I haven't checked extensively, but I think the rest of the code uses `_cat` as a suffix instead of `_class`. It would be good to stick to that convention (unless I'm wrong about it).

Reply:
This comment applies globally to other changes in this PR.